python/guide/module/gluon/nn.md

```diff
@@ -245,12 +259,14 @@ chimera(x)
 
 ## Compilation
```

Comment:
Compilation to Hybridization? And link to the dedicated tutorial.

Reply:
Changed, there's a link to Hybridization tutorial below.